hoot-core-test/src/test/cpp/hoot/core/io/ServiceOsmApiDbReaderTest.cpp

```diff
@@ -194,6 +196,12 @@ class ServiceOsmApiDbReaderTest : public CppUnit::TestFixture
     //members that are out of bounds.
     CPPUNIT_ASSERT_EQUAL(5, (int)map->getRelations().size());
 
+    //Need to remove timestamps, otherwise they cause issues with the compare
+    QList<ElementAttributeType> types;
+    types.append(ElementAttributeType(ElementAttributeType::Timestamp));
+    RemoveAttributeVisitor attrVis(types);
+    map->visitRw(attrVis);
+
```

Comment:
That is very interesting that this is the only test that uses the `OsmApiDbReader` class that doesn't also use the `MapComparator` class.

Reply:
Agreed, it's strange. It works... but maybe I should change it? What's your vote?